src/open_inwoner/cms/plugins/cms_plugins/tasks.py

```diff
@@ -0,0 +1,62 @@
+from django.utils.translation import gettext as _
+
+from cms.plugin_base import CMSPluginBase
+from cms.plugin_pool import plugin_pool
+from objectsapiclient.models import Configuration
+
+from ..models import TasksConfig
+
+
+@plugin_pool.register_plugin
+class TasksPlugin(CMSPluginBase):
+    """
+    Uses the Objects API to retrieve and show tasks according to the MijnTaken Objecttypes schema
+    Reuses the UserFeedPlugin template
+    """
+
+    model = TasksConfig
+    name = _("Task list Plugin")
+    render_template = "cms/plugins/tasks/tasks.html"
+    cache = False
+
+    def render(self, context, instance, placeholder):
+        request = context["request"]
+        context["instance"] = instance
+        context["tasks"] = []
+
+        if not request.user.is_authenticated or not (bsn := request.user.bsn):
+            return context
+
+        task_objects = self.get_tasks_by_bsn(user_bsn=bsn)
+        tasks = []
+        for obj in task_objects:
+            task_dict = obj.record["data"]
+            task_dict["task_url"] = obj.url
+            tasks.append(task_dict)
+        context["tasks"] = tasks
+
+        return context
+
+    def get_tasks(self):
+        objects_api_client = Configuration.get_solo().client
+        return [
+            obj
+            for obj in objects_api_client.get_objects(
+                object_type_uuid=self.model.object_type
+            )
+        ]
```

Comment:
```suggestion
        return [
           objects_api_client.get_objects(
                object_type_uuid=self.model.object_type
            )
        ]
```

Maybe it's already a list and `[...]` is not necessary?